content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -310,7 +331,7 @@ In version 4.1.0 and earlier, you cannot configure the transaction isolation lev
 
 </Admonition>
 
-To set the transaction isolation level, use the `isolationLevel` option in the second parameter of the API. For example:
+To set the transaction isolation level, use the `isolationLevel` option in the second parameter of the API. For example, for an interactive transaction:
```

Comment:
Would it make sense to add another code snippet for sequential transactions?

Reply:
Yes, that would be better. I'll have a stab at this.

Reply:
Thanks. I've added a code example now for sequential transactions.